hypervisor/src/hypervisor/config.rs

```diff
@@ -38,6 +39,20 @@ pub struct Partition {
     pub devices: Vec<Device>,
     #[serde(default)]
     pub hm_table: PartitionHMTable,
+    #[serde(default)]
+    pub udp_ports: Vec<SockAddr>,
+}
+
+/// A port the partition should listen on.
+/// A UDP socket will be created for this, that is bound to the specified port.
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct SockAddr(pub String);
```

Comment:
I personally dislike this name, due to a similarity with the POSIX struct. Maybe something like `UDPSock`?